src/main/java/dk/cngroup/lentils/service/ImageService.java

```diff
@@ -0,0 +1,36 @@
+package dk.cngroup.lentils.service;
+
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.File;
+import java.io.IOException;
+
+@Service
+public class ImageService {
+
+    public static final String IMAGE_DIRECTORY = "\\images\\";
```

Comment:
Tohle bych dal urcite nekde do konfirgracniho souboru, uz i s ohledem na testy, aby to tam slo prizpusobit.

Reply:
Souhlasim a kdyztak bych poprosil jeste podslozku "upload/". Tak aby to bylo "upload/images/". (Samozrejme na windowsech mas opacne lomitka. ;))